src/PlayableCharacter.java

```diff
@@ -0,0 +1,71 @@
+
+public abstract class PlayableCharacter extends Character{
+
+	//Atributes
+	private static byte nextlevelXP = 100; 	//Used for levelingUp
+	private int experience;					//Current XP, can't exceed experience for next level.
+	private Weapon equipedWeapon;
+	private Armor equipedArmor;
+	
```

Comment:
Remember to add the getters and setters, override toString and equals methods (use super method in the override)